### PR TITLE
Claim review crud and form

### PR DESCRIPTION
**Pull request by thesocialdev**
TODO: tests are failing and need to be fixed

### CODE REVIEW

File: packages/front/src/components/Claim.jsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 import {BrowserRouter, Switch, Route} from 'react-router-dom'
-import SpeechView from './SpeechView'
-import SpeechCreate from './SpeechCreate'
+import ClaimView from './ClaimView'
+import ClaimCreate from './ClaimCreate'
 
-const Speech = ({match}) => {
-    console.log("Speech", match.params.id);
+const Claim = ({match}) => {
+    console.log("Claim", match.params.id);
```
Comment (DuardoN) [inline packages/front/src/components/Claim.jsx]: Can remove 'console.log()'. This is not necessary.

File: packages/front/src/components/ClaimCreate.jsx [hunk 2]
```diff
@@ -5,26 +5,26 @@ import { Editor, EditorState } from 'draft-js'
...
-    saveSpeech(e) {
+    saveClaim(e) {
         e.preventDefault();
 
         const html = stateToHTML(this.state.editorState.getCurrentContent());
         const title = this.state.title;
         const personality = this.props.match.params.personalityId;
         console.log({ title, html, personality });
-        axios.post('http://localhost:3000/speech', { title, html, personality })
+        axios.post('http://localhost:3000/claim', { title, html, personality })
```
Comment (DuardoN) [inline packages/front/src/components/ClaimCreate.jsx]: We can remove this and others requester for folder where there happening requesters by methods. What do you thing?

Comment (thesocialdev) [reply packages/front/src/components/ClaimCreate.jsx]: Do you mean using a base URL?

File: packages/front/src/components/Claim.jsx (hunk 1 above)
Comment (thesocialdev) [reply packages/front/src/components/Claim.jsx]: Agreed. The next patch will have this removed.

File: packages/front/src/components/ClaimCreate.jsx (hunk 2 above)
Comment (DuardoN) [reply packages/front/src/components/ClaimCreate.jsx]: Yes.

**Pull request closed without merging**